news/_posts/2020-05-13-the-new-order-pages-in-prestaShop-1.7.7.md

```diff
@@ -0,0 +1,225 @@
+
+
+
+
+- **Editable order status labels:** Allows to change the order status directly from the listing.
+ 
+![Edit a status](/assets/images/2020/05/editable_order_statuses_labels.png "Edit a status")
+
+***What is the goal***:
+
+It allows the merchant fast management of the order fulfilment.
+
+***How it works***: 
+
+When the user clicks then on the status label, it displays a drop-down list with all of the available statuses. By selecting one of the items on the list, the user can update the order status.
+
+
+- **Statuses' colour**: Color according to the order status
+
+![Order statuses by category](/assets/images/2020/05/order_status_labelsx2.png "Order statuses by category")
+
+***What is the goal***: 
+
+The colours used by default to contextualize the status is based on a wide range of colours without following any real logic. To bring a structure and meaning to the statuses colours, the statuses are grouped in 4 colours categories:
+
+- orders waiting for a customer action as in waiting for a bank check in dark blue
+- orders waiting for a merchant action following an accepted payment in light blue
+- when no action is needed from the merchant as when the product is delivered in green
+- special status as payment error order in red or black
+
+The merchants have a concise view of the work remaining to be done.
+
+***How it works***: 
+
+The statuses colour is set by default. They are still configurable in the Order Setting page.
+
+
+- **Link to the customer's profile**: Creates easy access to the customer's profile by clicking on its name.
+
+![Right click on the customer's link](/assets/images/2020/05/link-customer.png "Right click on the customer's link")
+
+***What is the goal***: 
+
+Before the merchant used to have to open the order to get access to the customer page. Now, the merchant can do it from the orders listing.
+
+***How it works***:
+
+Clicking on the customer name opens the customer page in a new tab.
+
+
+
+
+- **Bulk action redesign** : The bulk actions' design and its position were updated to make it more visible and quicker to use.
+
+![Change_order's statuses](/assets/images/2020/05/choose_an_order.png "Change_order's statuses")
+
+_Updates the last 4 orders into payment accepted_
+
+***What is the goal***: 
+
+To help the merchants to process multiple orders in 3 clicks.
+
+***How it works***: 
+
+Select several orders, choose the new order status in the pop-in then validate it to update the status on the orders.
+
+
+
+
+- **Open in new tabs**: Open commands in new tabs by mass action to process them as the merchant proceeds.
+
+![Open in new tabs](/assets/images/2020/05/open_in_new_tabs.png "Open in new tabs")
+
+_Opens the last 4 orders in new tabs_
+
+***What is the goal***: 
+
+It helps the merchants to open multiple orders in 3 clicks.
```

Comment:
![image](https://user-images.githubusercontent.com/25405887/81817065-6f804b80-952c-11ea-867c-3c28d77861fc.png)

3 clicks again ???
No need to explain the goal here ^^